packages/tailwindcss-x/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "tailwindcss-x",
+  "private": true,
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "autoprefixer": "^10.4.4",
```

Comment:
I see that we are using the version `~9.7.4` can we keep both projects in sync? This way we will not have a package-lock.json here.